<?xml version="1.0" encoding="UTF-8"?>
<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
    <layouts>
        <layout>Error Layout</layout>
        <sections>
            <label>Liens personnalisés</label>
            <section>Custom Links</section>
        </sections>
    </layouts>
    <nameFieldLabel>Numéro d&apos;erreur</nameFieldLabel>
    <standardFields>
        <name>Name</name>
        <label>Numéro d&apos;erreur</label>
    </standardFields>
    <gender>Feminine</gender>
    <startsWith>Vowel</startsWith>
    <caseValues>
        <plural>false</plural>
        <value>Erreur</value>
    </caseValues>
    <caseValues>
        <plural>true</plural>
        <value>Erreurs</value>
    </caseValues>
    <fields>
        <name>Context_Type__c</name>
        <help>Contexte ayant généré l’erreur, s’il est connu.</help>
        <label>Type de contexte</label>
    </fields>
    <fields>
        <name>Datetime__c</name>
        <help>Date et heure auxquelles l’erreur s’est produite</help>
        <label>Date et heure</label>
    </fields>
    <fields>
        <name>Email_Sent__c</name>
        <help>Indique si une notification par e-mail concernant cette erreur a été envoyée.</help>
        <label>E-mail envoyé</label>
    </fields>
    <fields>
        <name>Error_Type__c</name>
        <help>Type de l’erreur survenue</help>
        <label>Type d’erreur</label>
    </fields>
    <fields>
        <name>Full_Message__c</name>
        <help>Texte complet du message d’erreur</help>
        <label>Message complet</label>
    </fields>
    <fields>
        <name>Object_Type__c</name>
        <help>Type de l’objet sur lequel l’erreur s’est produite, s’il est connu.</help>
        <label>Type d’objet</label>
    </fields>
    <fields>
        <name>Posted_in_Chatter__c</name>
        <help>Indique si l’erreur a été publiée dans Chatter.</help>
        <label>Publiée dans Chatter</label>
    </fields>
    <fields>
        <name>Record_URL__c</name>
        <help>Lien vers l’enregistrement ayant provoqué l’erreur, si disponible.</help>
        <label>URL d’enregistrement</label>
    </fields>
    <fields>
        <name>Related_Record_ID__c</name>
        <help>L&apos;ID Salesforce de l&apos;enregistrement associé à cette erreur.</help>
        <label>ID d&apos;enregistrement associé</label>
    </fields>
    <fields>
        <name>Retry_Pending__c</name>
        <help>Lorsque cette option est sélectionnée, l&apos;enregistrement spécifié dans ID d&apos;enregistrement associé est retraité dans le processus suivant. Si le nouveau traitement réussit, ce champ est désélectionné. S&apos;applique aux types d&apos;erreur des processus planifiés.</help>
        <label>Nouvelle tentative en attente ?</label>
    </fields>
    <fields>
        <name>Stack_Trace__c</name>
        <help>Trace de pile pour l’erreur déclenchée, si disponible dynamiquement.</help>
        <label>Trace de pile</label>
    </fields>
</CustomObjectTranslation>
